packages/driver/src/cypress/commands.ts

```diff
@@ -13,6 +13,10 @@ const builtInCommands = [
   addCommand,
 ]
 
+const reservedCommandNames = [
```

Comment:
This list is going to grow, so better to make it an object. That way the lookup is O(1) instead of O(n).

Reply:
@chrisbreiding So would this be considered a breaking change, considering the fact that using these reserved names currently only causes an error under specific circumstances?

Reply:
Agreed this should be considered breaking. Changes don't have to be large or affect many people in order to be considered breaking - they just have to render previously working code now troublesome.